server.go

```diff
@@ -147,15 +146,15 @@ func (s *Server) handleConn(c *Conn) error {
 // ListenAndServe listens on the network address s.Addr and then calls Serve
 // to handle requests on incoming connections.
 //
-// If s.Addr is blank and LMTP is disabled, ":smtp" is used.
```

Comment:
This change doesn't belong to this PR.

Reply:
If we dont support TCP for lmtp, then the README.md example with telnet is wrong.

Reply:
Why would it be wrong? The current README example creates a TCP server.

Reply:
With my patch i have enabled TCP mode for LMTP, this was not allowed before and was forced to unixsocket. 
I have created an example for a LMTP TCP server with telnet as client in README.md, this example would be wrong if we dont accept TCP Mode for lmtp here.